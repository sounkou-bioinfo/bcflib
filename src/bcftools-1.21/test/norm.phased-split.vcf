##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=20,length=2147483647>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
20	1	.	A	C,G,T	.	.	.	GT	3|2|1|0
