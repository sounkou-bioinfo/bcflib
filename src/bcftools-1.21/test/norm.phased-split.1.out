##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=20,length=2147483647>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
20	1	.	A	C	.	.	.	GT	0|0|1|0
20	1	.	A	G	.	.	.	GT	0|1|0|0
20	1	.	A	T	.	.	.	GT	1|0|0|0
